bundles/org.openhab.binding.epsonprojector/src/main/feature/feature.xml

```diff
@@ -5,6 +5,7 @@
 	<feature name="openhab-binding-epsonprojector" description="Epson Projector Binding" version="${project.version}">
 		<feature>openhab-runtime-base</feature>
 		<feature>openhab-transport-serial</feature>
+		<feature>openhab-core-config-discovery-sddp</feature>
```

Comment:
Because it is not part of openhab-runtime-base ?

Reply:
Correct.